projects/elife.yaml

```diff
@@ -2099,3 +2099,17 @@ data-pipeline:
                  deletion-policy: delete
             "{instance}-elife-data-pipeline-archive":
                  deletion-policy: retain
+    gcp:
+        bigquery:
+            "{instance}": # dataset
+                # does 'project' belong beneath dataset? do we want an instance affecting multiple projects?
+                project: elife-data-pipeline
+                tables:
+                    # bit of repetition here
+                    380_datascience_early_career_researchers:
```

Comment:
storing the schema here is supported, but if the project needs to change it often (as in every deploy), it should be able to not specify one; not sure how a migration process works for BigQuery?

Reply:
I was thinking the flow would ideally be like this: 

1. remote schema changes
2. ./bldr update_infrastructure:stackname
3. terraform is given a remote path to the schema, pulls it down, and tells you if changes are to be made
4. terraform applies changes

we're not managing the schema of other databases through builder, we leave it to the application typically. I wonder if we should at the dataset level - just ensure the project+dataset exists and let the app deal with table creation

Reply:
or maybe we just create and delete tables and that's it, no updates

Reply:
We can create the dataset only if the project requires it, it's already supported. If the schema changes are so involved though then having them in `builder` is not a big limitation, as they need coordination during deployments anyway.